gms/feature_service.hh

```diff
@@ -138,6 +138,7 @@ public:
     gms::feature group0_schema_versioning { *this, "GROUP0_SCHEMA_VERSIONING"sv };
     gms::feature supports_consistent_topology_changes { *this, "SUPPORTS_CONSISTENT_TOPOLOGY_CHANGES"sv };
     gms::feature host_id_based_hinted_handoff { *this, "HOST_ID_BASED_HINTED_HANDOFF"sv };
+    gms::feature service_levels_raft_notifications { *this, "SERVICE_LEVELS_RAFT_NOTIFICATIONS"sv };
```

Comment:
Do you really need a separate feature here or can just wire it up to the overall topology-on-raft bit?

Reply:
> Do you really need a separate feature here or can just wire it up to the overall topology-on-raft bit?

If this will be included 6.0 then yes, we could reuse the existing feature, but this PR does not assume that this will happen.